Makefile

```diff
@@ -5,10 +5,10 @@ TARGETS = naive cuda cupla kokkos alpakaCT alpakaRT #CT: compile-time, RT:
 TBB_BASE := /data/cmssw/slc7_amd64_gcc820/external/tbb/2019_U3-pafccj
 BOOST_BASE := /data/cmssw/slc7_amd64_gcc820/external/boost/1.67.0-pafccj/
 CXX := g++
-CXX_FLAGS := -O2 -std=c++14 -ftemplate-depth-512 -I$(BOOST_BASE)/include -I$(TBB_BASE) -L$(TBB_BASE)/lib 
+CXX_FLAGS := -O2 -std=c++14 -ftemplate-depth-512 -I$(BOOST_BASE)/include 
 CXX_DEBUG := -g
 NVCC := /usr/local/cuda/bin/nvcc -ccbin $(CXX)
-NVCC_FLAGS := -O2 -std=c++14 --expt-relaxed-constexpr -w --generate-code arch=compute_35,code=sm_35 -I$(BOOST_BASE)/include
+NVCC_FLAGS := -O2 -std=c++14 --expt-relaxed-constexpr -w --generate-code arch=compute_75,code=sm_75 -I$(BOOST_BASE)/include
```

Comment:
Architecture ```sm_35``` does not work for me on GPU GeForce RTX 2080, I got error at the run-time.
```
cudaErrorNoKernelImageForDevice: no kernel image is available for execution on the device
```

Reply:
> Architecture `sm_35` does not work for me on GPU GeForce RTX 2080, I got error at the run-time.
> 
> ```
> cudaErrorNoKernelImageForDevice: no kernel image is available for execution on the device
> ```

Yes, that needs 70 or 75.
Maybe we should find a way to discover the "local" architecture at runtime ?